### PR TITLE
Classify endpoint that sorts images into the proper recycling bin

### PR DESCRIPTION
**Pull request by fairnightzz**

### CODE REVIEW

File: internal/sorter/api.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+						MaxResults: 15,
+					},
+				},
+			},
+		},
+	}
+
+	payload, err := json.Marshal(imagePayload)
+
+	if err != nil {
```
Comment (MrPicklePinosaur) [inline internal/sorter/api.go:47]: these should return status 500s

File: internal/sorter/api.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/gorilla/mux"
+)
+
+func ClassifyHandler(w http.ResponseWriter, r *http.Request) {
```
Comment (MrPicklePinosaur) [inline internal/sorter/api.go:17]: would be nice to seperate some logic out to other functions

File: internal/sorter/api.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	}
+	response.Type = found
+
+	jsonResponse, err := json.Marshal(response)
+	if err != nil {
+		return
+	}
+
+	conn, err := db.Connection()
+	uId, err := db.GetUserId(conn, userId)
```
Comment (MrPicklePinosaur) [inline internal/sorter/api.go]: go convention is to capitalize acroynms like ID

File: internal/sorter/api.go [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+	if err = json.Unmarshal(body, &m); err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	// sb := string(body)
+
+	var response entity.ClassifyResponse
+
+	blueBinPossible, err := getBlueBin()
```
Comment (MrPicklePinosaur) [inline internal/sorter/api.go:77]: handle these errors

File: internal/sorter/api.go [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+	conn, err := db.Connection()
+	uId, err := db.GetUserId(conn, userId)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	itemRecord := db.Item{UserID: uId, Type: found, Name: identified}
+	err = db.AddItem(conn, &itemRecord)
+	if err != nil {
+		fmt.Println(err)
```
Comment (MrPicklePinosaur) [inline internal/sorter/api.go]: forgot to return?

File: internal/sorter/file.go [hunk 6]
```diff
@@ -0,0 +1,38 @@
+package sorter
+
+import (
+	"errors"
+	"io/ioutil"
+	"strings"
+)
+
+func getBlueBin() (map[string]bool, error) {
```
Comment (MrPicklePinosaur) [inline internal/sorter/file.go:9]: `getRedBin` and `getBlueBin` can be the same function, pass in the file path as a parameter

File: pkg/db/db.go [hunk 7]
```diff
@@ -35,13 +36,40 @@ func Migrate(conn *gorm.DB) error {
 			&Item{},
 		)
 		if err != nil {
+			log.Fatal(err)
```
Comment (MrPicklePinosaur) [inline pkg/db/db.go]: sus

**Review by MrPicklePinosaur — CHANGES_REQUESTED** [review]

File: internal/sorter/api.go (hunk 2 above)
Comment (fairnightzz) [reply internal/sorter/api.go:17]: 🗿

**Review by MrPicklePinosaur — APPROVED** [review]
lgtm

**Pull request merged**